api-flask/README.md

```diff
@@ -18,6 +18,30 @@ Exposes a sample API response and takes the following query arguments:
 - `?group_by`, a key to use to group zones in the geojson, eg. `ADM1_PCODE`
 - `?geojson_out`, decide if the output should be a geojson or a list of data. Default is false -> List.
 
+### `/alerts-all` (GET)
+Return all the alerts data that `alert` table holds
+```
+curl --location --request GET 'localhost:80/alerts-all' > data.json
+```
+
+### `/alerts` (GET)
+Based on the parameter from reuest URL, this endpoint will return the matched
+alert rows from DB.
+- `id` return alert data that has `id`
+- TODO: more GET all operations will be supported for different query cases
+```
+curl --location --request GET 'localhost:80/alerts?id=3'
+```
```

Comment:
A more "tradition" REST way to make these urls would be to do something like

```
GET /alerts/123   <--- GET single alert id#123
GET /alerts       <--- GET all alerts
```

Not sure how you could represent that in Flask off the top of my head. The way you have it is (in my opinion) also fine.

Reply:
Agree, it would be `/alerts/{id}`. This operation here is just an entry point for more structured query.

Since `id` is an internal primary key for `alerts`, I don't think we will use it for query in the future.

Reply:
Agree with @rysavym-amzn let's keep the endpoints "traditional"

Reply:
sure, i will update this endpoint to follow `/alerts/{id}` convention.